<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyTables</class>
 <widget class="QMainWindow" name="modifyTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="cb_table"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_download">
        <property name="text">
         <string>Alle herunterladen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_companyName"/>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_gold">
        <property name="text">
         <string>Gold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_silver">
        <property name="text">
         <string>Silber</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_bronze">
        <property name="text">
         <string>Bronze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_supporter">
        <property name="text">
         <string>Supporter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="tv_table"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_save">
        <property name="text">
         <string>Ã„nderungen speichern</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
